docs/program/_index.md

```diff
@@ -41,9 +41,7 @@ Next, to install your preferred Viam SDK on your Linux or macOS development mach
 {{< tabs >}}
 {{% tab name="Python" %}}
 
-```sh {class="command-line" data-prompt="$"}
```

Comment:
but we want to keep this command - just put it after the virtual env

Reply:
ok done!!